_source/_posts/2022-07-21-jhipster-ci-cd.md

```diff
@@ -0,0 +1,865 @@
+---
+layout: blog_post
+title: "Continous integration and delivery for JHipster microservices"
```

Comment:
```suggestion
title: "Continous Integration and Delivery for JHipster Microservices"
```
we are still using "Title Case" for the main title, for all the section headings we are using "Sentence case"